server/grpc_service.go

```diff
@@ -303,10 +303,14 @@ func (s *Server) RegionHeartbeat(stream pdpb.PD_RegionHeartbeatServer) error {
 			return errors.Trace(err)
 		}
 
+		storeID := request.GetLeader().GetStoreId()
+		storeLabel := fmt.Sprintf("tikv%d", storeID)
```

Comment:
I think you can remove tikv here.

Reply:
I think `tikv42` is more meaningful than `42` when it shows on grafana page.

Reply:
We can configure how to show the name in Grafana directly, like `tikv-{store}`.

Reply:
ok.